pod.go

```diff
@@ -269,6 +269,12 @@ type PodConfig struct {
 	// This list can be empty and populated by adding containers
 	// to the Pod a posteriori.
 	Containers []ContainerConfig
+
+	// Annotations is a reserved field and can be filled with various
+	// different data by each user. This means that virtcontainers will
+	// never try to interpret those data, it is only providing a way to
+	// save some data for users.
```

Comment:
We need to be more precise on how `Annotations` should be used as there's nothing preventing different users to add their own annotations to the same pod config.
So we need to add something like:
```
Annotations keys must be unique strings an must be name-spaced with e.g. reverse domain notation (org.clearlinux.key).
```